am-lib/src/main/java/uk/gov/hmcts/reform/amlib/internal/utils/SecurityClassifications.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.amlib.internal.utils;
+
+import uk.gov.hmcts.reform.amlib.enums.SecurityClassification;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+import static java.util.stream.Collectors.toSet;
+
+public final class SecurityClassifications {
+
+    private SecurityClassifications() {
+        //NO-OP
```

Comment:
Shall we throw an exception, it seemed to be more recent pattern.

Reply:
have updated along with Permissions.class